src/arm.rs

```diff
@@ -8,7 +8,7 @@ pub unsafe fn __aeabi_uidivmod() {
     asm!("push {lr}
           sub sp, sp, #4
           mov r2, sp
-          bl __udivmodsi4
+          bl ___udivmodsi4
```

Comment:
Apparently the `gnueabi` targets **don't** need the extra underscore... (while the iOS versions of those targets **do**).